README.md

```diff
@@ -30,6 +30,18 @@ Todo:
 - (TBD) [Alveo U50](https://docs.amd.com/r/en-US/ug1371-u50-reconfig-accel)
 - (TBD) [Alveo U280](https://docs.amd.com/r/en-US/ug1314-alveo-u280-reconfig-accel)
 
+# Build and Run:
+The top-level Makefile can be used to build the system-on-chip for the specific target board.
+First, define environmental variables by using:
+```
+source settings.sh
+```
+Then, download rtl sources for non-xilinx IPS and build the SoC by running:
+```
+make 
```

Comment:
Split the steps one by one:
1. Setup -> source settings.sh
2. Fetch sources -> make units
3. Build bitstream -> make xilinx